index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    
+    <header>
+        <nav class="menu_icon">
+                <div></div>
+                <div></div>
+                <div></div>
```

Comment:
- [ ] Great work using the `<nav>` bar to wrap your burger menu 💯. However, please kindly ensure to remove all empty `<div>`  tags. 🤓